.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,25 @@
+name: Dependabot auto-merge
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
```

Comment:
Interesting `[bot]` syntax - I guess that's how GitHub exposes the actor name, to prevent someone just creating an account called 'dependabot'?
Also looks like `actor` is the right one to use [instead of `triggering_actor`](https://docs.github.com/en/actions/learn-github-actions/contexts) 👍